app/actions/views/channel.js

```diff
@@ -627,9 +627,10 @@ function loadGroupData() {
                             });
                         }
                     } else {
+                        const getGroupsSince = isReconnect ? (state.websocket?.lastConnectAt || 0) : undefined;
                         const [getAllGroupsAssociatedToChannelsInTeam, getGroups] = await Promise.all([ //eslint-disable-line no-await-in-loop
                             Client4.getAllGroupsAssociatedToChannelsInTeam(team.id, true),
-                            Client4.getGroups(true, 0, 0),
+                            Client4.getGroups(true, 0, 0, getGroupsSince),
```

Comment:
```suggestion
                            Client4.getGroups(true, 0, 0, state.websocket?.lastConnectAt),
```
If it's null or 0, your params default to 0 which then means no since will be passed into the backend endpoint

Reply:
`state.websocket?.lastConnectAt` value would be with current time stamp eg: `1612882888370`.  So it will affect the first time loading.